### PR TITLE
mynewt: adjust timings in MESH/NODE/CFG/HBP/BV-05-C

### PR DESCRIPTION
**Pull request by KKopyscinski**

### CODE REVIEW

File: ptsprojects/mynewt/mesh.py [hunk 1]
```diff
@@ -42,8 +42,8 @@
 from time import sleep
 
 
-device_uuid = hexlify(uuid4().bytes)
-device_uuid2 = hexlify(uuid4().bytes)
+device_uuid = "00000000000000000000000000000000"
```
Comment (michal-narajowski) [inline ptsprojects/mynewt/mesh.py:45]: Why is this change here?

**Review by michal-narajowski — CHANGES_REQUESTED** [review]

File: ptsprojects/mynewt/mesh.py (hunk 1 above)
Comment (KKopyscinski) [reply ptsprojects/mynewt/mesh.py:45]: Is there a reason that for Mynewt these are random?

**Review by michal-narajowski — APPROVED** [review]
